plugin.yaml

```diff
@@ -651,6 +651,10 @@ data_types:
         type: string
         description: The Availability Zone for the subnet.
         required: false
+      UseAvailableZones:
+        type: boolean
+        description: A boolean to choose another available zone if the one provided is not available.
+        required: false
```

Comment:
`resource_config` is reserved for API keywords, this should be a property. And properties are all lower case separated by `_`.